mlflow/recipes/utils/wrapped_recipe_model.py

```diff
@@ -0,0 +1,35 @@
+import mlflow
+from mlflow.pyfunc import PythonModel
+import pandas as pd
+import numpy as np
+
+
+class WrappedRecipeModel(PythonModel):
+    def __init__(self, predict_scores_for_all_classes=True, prefix="predicted_"):
```

Comment:
Do `predict_scores_for_all_classes` and `prefix` need to have default value? It should be set by the caller to make sure the value is passed down correctly.